components/x-live-blog-wrapper/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@financial-times/x-live-blog-wrapper",
+  "version": "0.0.0",
+  "description": "",
+  "main": "dist/LiveBlogWrapper.cjs.js",
+  "module": "dist/LiveBlogWrapper.esm.js",
+  "browser": "dist/LiveBlogWrapper.es5.js",
+  "scripts": {
+    "build": "node rollup.js",
+    "start": "node rollup.js --watch"
+  },
+  "keywords": [
+    "x-dash"
+  ],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@financial-times/x-engine": "file:../../packages/x-engine",
+    "@financial-times/x-live-blog-post": "file:../x-live-blog-post",
+    "@financial-times/x-interaction": "file:../x-interaction"
+  },
+  "devDependencies": {
+    "@financial-times/x-rollup": "file:../../packages/x-rollup",
+    "@financial-times/x-test-utils": "file:../../packages/x-test-utils"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Financial-Times/x-dash.git"
+  },
+  "homepage": "https://github.com/Financial-Times/x-dash/tree/master/components/x-live-blog-wrapper",
+  "engines": {
+    "node": ">= 6.0.0"
```

Comment:
We prob wanna update the node version here or we can remove the field altogether